tests/php/Model/SiteTreeTest.php

```diff
@@ -888,7 +887,7 @@ public function testAuthorIDAndPublisherIDFilledOutOnPublish()
             "SELECT \"AuthorID\", \"PublisherID\" FROM \"SiteTree_Versions\"
             WHERE \"RecordID\" = ? ORDER BY \"Version\" DESC",
             [$about->ID]
-        )->first();
+        )->record();
```

Comment:
I'm surprised this used to work

Reply:
Should I return back a method `first()`?

Reply:
No it's correct, first() was just a deprecated alias for record() - sorry I should have checked that before commenting